README.org

```diff
@@ -14,21 +14,25 @@
 
 ** Synopsis
 
-=org-journal= maintains a set of files, where each file represents a day. Convenient
-bindings allow the creation of journal records in the current daily file and search within
-all records or specified time intervals. All records can be browsed and searched from the
-Emacs Calendar for convenience. All entries in a specified TODO state will be carried over
-to the next day. Optionally, the journal can also be encrypted.
-
-An example of a daily file (it will actually look a lot nicer, depending on your org-mode
+=org-journal= maintains a set of files, where each file represents a
+day. Convenient bindings allow the creation of journal records in the current
```

Comment:
Do you want to add something here? Like, "each file represents a day (or week, month, year)".

Reply:
Makes sense. I will add something.